crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/declare.rs

```diff
@@ -73,11 +74,14 @@ fn get_contract_class(contract_artifact: &StarknetContractArtifacts) -> Runnable
     )
     .expect("Failed to read contract class from json");
 
-    match &contract_artifact.executor {
+    #[cfg(feature = "cairo-native")]
+    return match &contract_artifact.executor {
         None => RunnableCompiledClass::V1(contract_class),
         Some(executor) => RunnableCompiledClass::V1Native(NativeCompiledClassV1::new(
```

Comment:
Can't this be as follows?
```suggestion
    return match &contract_artifact.executor {
        None => RunnableCompiledClass::V1(contract_class),
        #[cfg(feature = "cairo-native")]
        Some(executor) => RunnableCompiledClass::V1Native(NativeCompiledClassV1::new(
```

Reply:
It cannot because `executor` field itself is behind a feature flag as well.